hs_core/tasks.py

```diff
@@ -343,23 +348,29 @@ def create_temp_zip(resource_id, input_path, output_path, sf_aggregation, sf_zip
             res.create_aggregation_xml_documents()
 
     try:
-        if sf_zip:
+        if aggregation or sf_zip:
             # input path points to single file aggregation
             # ensure that foo.zip contains aggregation metadata
             # by copying these into a temp subdirectory foo/foo parallel to where foo.zip is stored
             temp_folder_name, ext = os.path.splitext(output_path)  # strip zip to get scratch dir
             head, tail = os.path.split(temp_folder_name)  # tail is unqualified folder name "foo"
             out_with_folder = os.path.join(temp_folder_name, tail)  # foo/foo is subdir to zip
             istorage.copyFiles(input_path, out_with_folder)
-            if sf_aggregation:
+            if aggregation:
                 try:
-                    istorage.copyFiles(input_path + '_resmap.xml',  out_with_folder + '_resmap.xml')
+                    istorage.copyFiles(aggregation.map_file_path,  temp_folder_name)
                 except SessionException:
-                    logger.error("cannot copy {}".format(input_path + '_resmap.xml'))
+                    logger.error("cannot copy {}".format(aggregation.map_file_path))
                 try:
-                    istorage.copyFiles(input_path + '_meta.xml', out_with_folder + '_meta.xml')
+                    istorage.copyFiles(aggregation.metadata_file_path, temp_folder_name)
                 except SessionException:
-                    logger.error("cannot copy {}".format(input_path + '_meta.xml'))
+                    logger.error("cannot copy {}".format(aggregation.metadata_file_path))
+                for file in aggregation.files.all():
+                    logger.error(file)

```

Comment:
should be logger.info rather than logger.error above, or just remove the line after debugging phase is done

Reply:
that's leftover from debugging, removed.